advanced/attributes/src/main/java/org/arakhne/afc/attrs/attr/AttributeType.java

```diff
@@ -152,53 +155,118 @@
 		NAME_RESOURCE_FILE = pName + ".types"; //$NON-NLS-1$
 	}
 
+	/** FactoryMap collection.
+	 *
+	 */
+	private static final Map<AttributeType, StringFactory> FACTORY_MAP =
+			Collections.unmodifiableMap(new HashMap<AttributeType, StringFactory>() {{
```

Comment:
Thank you for making this clear.

On Thu, Jun 16, 2016 at 1:07 PM, Stéphane Galland <notifications@github.com>
wrote:

> In
> advanced/attributes/src/main/java/org/arakhne/afc/attrs/attr/AttributeType.java
> <https://github.com/gallandarakhneorg/afc/pull/138#discussion_r67319466>:
>
> > @@ -152,53 +155,118 @@
> >  		NAME_RESOURCE_FILE = pName + ".types"; //$NON-NLS-1$
> >  	}
> >
> > +	/** FactoryMap collection.
> > +	 *
> > +	 */
> > +	private static final Map<AttributeType, StringFactory> FACTORY_MAP =
> > +			Collections.unmodifiableMap(new HashMap<AttributeType, StringFactory>() {{
>
> Ok, I understand. The first block is for the type definition, and the
> second inner block is the instance initialisation block. It's ok for me.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/gallandarakhneorg/afc/pull/138/files/8e6fe6d85c4de5314d335bd8e78fe4be9c085576#r67319466>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AR09BOhCFpTftWy4mOr95Dx00zvS-_jBks5qMSBXgaJpZM4I2s4F>
> .
>